release.Rmd

```diff
@@ -26,21 +63,304 @@ To get your package ready to release, follow these steps:
+For the tidyverse team specifically, it also feels appropriate for us as full-time package maintainers to meet other maintainers more than halfway.
+If you decide to do this, make a new issue or a new checklist in the release issue that lists all the packages that you need to fix.
+For example, [here](https://github.com/tidyverse/dplyr/issues/6262) is the issue where dplyr maintainers tracked hundreds of PRs in the build-up to releasing dplyr v1.1.0.
+As you create the PRs, add links to those as well.
+As the revdep maintainers merge the PRs, check them off in your list.
+You should let the maintainers know when you plan to submit to CRAN, so they can submit their updated version before that.
+But their submission is up to them - you can submit to CRAN knowing that you've done your due diligence.
+If you explain to CRAN that you gave advance notice and provided patches, they will accept your update, even if it breaks some other packages.
+
+If your update breaks so many packages that it's not feasible to provide PRs for all of them, you need to seriously reconsider whether the benefit of the change is worth it.
+If it affects that many packages, it's likely to also affect a lot of code that lives outside of CRAN packages, such as scripts, reports, and Shiny apps.
+If you still decide to go ahead (presumably because the change is *really* compelling), you can use `revdepcheck::revdep_email()` (or, in our case, `revdepcheck::cloud_email()`) to notify maintainers *en masse*.
+Make sure the email includes a link to documentation that describes the most common breaking changes and how to fix them.
+
+### Update comments for CRAN
+
+-   [ ] Update `cran-comments.md`
+
+We use the `cran-comments.md` file to record comments about a submission, mainly just the results from `R CMD check` and revdep checks.
+We like to track this file in Git, so we can see how it changes over time.
+It should also be listed in `.Rbuildignore`, since it should not appear in your package bundle.
+When you're ready to submit, `devtools::submit_cran()` (TODO: forward reference) incorporates the contents of `cran-comments.md` when it uploads your submission.
+
+The target audience for these comments is the CRAN personnel, although there is no guarantee that they will read the comments (or when in the submission process they read them).
+For example, if your package breaks other packages, you will likely receive an automated email about that, even if you've explained that in the comments.
+Sometimes a human at CRAN then reads the comments, is satisfied, and accepts your package anyway.
+At other times, your package may be stuck in the queue until you copy `cran-comments.md` and paste it into an email exchange to move things along.
+In any case, it's worth keeping these comments in its own, version-controlled file.
+
+Here are the `cran-comments.md` from a recent version of httr: TODO: update this, what's a good current example?
```

Comment:
https://github.com/tidyverse/tidyr/blob/main/cran-comments.md is another example

Reply:
I went with dplyr, to stick with the same example, since I linked to its issue tracking the many PRs made to update revdeps.